docs/getting-started/migrating-workflows-to-github-apps.rst

```diff
@@ -0,0 +1,111 @@
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+        - /test/dockstore.cwl.json
+
+The information above was filled out using the following:
+
+- ``subclass`` is taken from the ``Descriptor Type``
+- ``primaryDescriptorPath`` is from ``Workflow Path``
+- ``testParameterFiles`` is from ``Test File Path``
+
+During the original registration for your workflow, you may have filled out the ``Workflow Name`` field shown in the picture below.
+
+.. figure:: /assets/images/docs/workflow-name-field.png
+   :alt: Workflow to Migrate
+   :width: 60 %
+
+Typically this field is used when you want to register multiple workflows from the same repo, but you may have filled it out regardless. To check if the workflow
+you want to migrate has a workflow name set, select the workflow and look at the title on top as shown in the picture below.
+
+.. image:: /assets/images/docs/workflow-title.png
+
+The title consists of:
+``<sourceControl>/<organization name>/<repository name>/<optional workflow name>:<version name>``
+
+If you see a workflow name inserted, you must include the name field in your ``/.dockstore.yml``.
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+        - /test/dockstore.cwl.json
+        name: optional-name
+
+If you have multiple workflows stemming from the same repo, a single ``/.dockstore.yml`` can be used to convert them. You should also keep the information about
+the Workflow Name in mind because at least one of your workflows must have that field set. Let's say we want to convert these two workflows that come from this `repository <https://github.com/NatalieEO/ghapps-single-workflow>`__.
+
+.. image:: /assets/images/docs/github-apps-multiple-workflows.png
+
+.. image:: /assets/images/docs/github-apps-multiple-workflows-with-name.png
+
+Your ``/.dockstore.yml`` would look like the following:
+
+.. code:: yaml
+
+   version: 1.2
+   workflows:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+        - /test/dockstore.cwl.json
+      - subclass: WDL
+        primaryDescriptorPath: /Dockstore.wdl
+        testParameterFiles:
+        - /test/dockstore.wdl.json
+        name: optional-name
+
+Testing the Migration
+----------------------
+
+.. note:: Push events will only be captured by Dockstore **after** installing the GitHub app onto the repo.
+
+To test out your GitHub app integration, make a push to your master branch. Navigate to or refresh the My Workflows page and select the workflow you wanted to convert.
+You should see that the ``Workflow Information`` section looks a bit different.
+
+.. image:: /assets/images/docs/workflow-information-after-migration.png
```

Comment:
Does the versions table look different if it's a mixture of migrated and non-migrated? If so, do you want have a screenshot for that?

Reply:
Other than the actions dropdown having the refresh option greyed out for migrated versions, the table doesn't look different